_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -13,64 +13,72 @@ Security Analytics provides the options and functionality to monitor and respond
 
 You can define a new detector by naming the detector, selecting a data source and detector type, and specifying a detector schedule. After defining a detector, you can also configure field mappings and set up alerts. Follow the steps in this section to accomplish all three of these setup tasks.
 
-1. On the Detectors page, select the **Create detector** button. The Define detector page opens.
-1. Give the detector a name and, as an option, add a description for the detector. 
-1. In the Data source section, select the dropdown arrow and select a source for the log data.
-1. In the threat detection type section, select the data type. The Sigma security rules associated with the log data are automatically populated in the Detection rules section below it.
-<img src="{{site.url}}{{site.baseurl}}/images/Security/detector_rules.png" alt="Selecting threat detector type to auto-populate rules" width="80%">
+1. On the **Threat detectors** page, choose **Create detector**. The **Define detector** page opens.
+1. In **Detector details**, give the detector a name and, as an option, add a description for the detector. 
```

Comment:
This sentence is awkward to read. What about:

```
In **Detector details**, give the detector a name and optionally add a description.
```

or something like that? Or maybe there isn't a better wording?

Reply:
"... give the detector a name. Adding a description for the detector is optional."